labs/b5.md

```diff
@@ -0,0 +1,373 @@
+
+Some port numbers for well known services are as follows:
+Service     | Port
+------------|------
+HTTP        |80
+HTTPS       |443
+
+
+# Sysadmin Commands
+As a sysadmin, trying to diagnose network issues can often be pretty challenging. Given the scale and complexity of networks, it's tough trying to narrow down the scope of a problem to a point of failure. What follows is a list of commands/tools that can help with triaging problems. There are a lot of tools and we don't expect you to memorize every single detail. However, it is important to know what tools exist and when to use them when a problems inevitably arise. If you ever need more details the `man` pages for these commands are a great place to turn to for reference.
+
+Tools also tend to overlap in functionality -- for example there are multiple tools that can display interface information or test connectivity.  When possible, it is a good idea to use multiple tools to cross-check one another.
+
+Note that when it comes to real world networks there are even more factors to consider that we haven't touched on like network security. For example, two machines can have a fully functioning connection but if one machine has been configured to drop all packets then it might seem as if they aren't connected. 
+
+So take the output of these tools with a grain of salt, they a means of narrowing down issues. It is important not to misinterpret outputs or jump to conclusions too quickly.
+
+1. `hostname`
+A simple and straightforward command that can display information about a host, IP addresses, FQDN, and etc.  Make sure to also check out `host`, which is a similar command that provides more detailed information by doing a lookup on a given name.
+
+2. `ping`
+ Another simple command, most of the time you'll be using ping as a first step towards testing connectivity. If ping can't reach a host then there is likely an issue with connectivity. The ping tool does this by sending out ICMP messages to the host expecting a response. (More on the protocol [here][ICMP])
+
+    Moreover ping also provides metrics for Round Trip Time (RTT) and packet loss.  Round trip time is defined as the time it takes for a response to arrive after sending the ping packet.  These can prove to be very useful statistics.
+
+[ICMP]: https://en.wikipedia.org/wiki/Internet_Control_Message_Protocol
+
+3. `traceroute`
```

Comment:
This gets renumbered as `1` again instead of `3`, I think the links are messing it up as the same happens with netstat further down, so I think if you put the link references at the end of the list it'll be fine.